### ISSUE TITLE
.wav files greyed out and can not transfer to MZ-RH1 (Ubuntu 18.04 v0.9.15)

### ISSUE THREAD

**Issue by djsbriscoe** [linux-minidisc/linux-minidisc#44]
The wav files I want to transfer to my MZ-RH1 are visible but are greyed out. I can connect to the device and the files actually on the device are visible. I am not sure if this is a permissions issue. Is there a way of using the terminal to try to send files and get some kind of feedback or error messages? Also my wave files are around 50Mb each so I'm not sure if there is a size limit. What tests can I perform to try to correct this issue? Thanks.

**Comment by thp** [comment]
WAV is a container file format, can you check with e.g. `soxi` (in the [sox](https://packages.debian.org/stretch/sox) package) what the file contents are? Is there any debug output when running the GUI on the command line?

**Comment by djsbriscoe** [comment]
Sorry for the delay in replying.

Here is the output of Soxi for 2 files (copied from CD using EAC V1.3). These files are greyed out in QHiMDtransfer

Input File     : '/home/david/Music/TASW/Track02.wav'
Channels       : 2
Sample Rate    : 44100
Precision      : 16-bit
Duration       : 00:05:29.73 = 14541240 samples = 24730 CDDA sectors
File Size      : 58.2M
Bit Rate       : 1.41M
Sample Encoding: 16-bit Signed Integer PCM

Input File     : '/home/david/Music/TASW/Track17.wav'
Channels       : 2
Sample Rate    : 44100
Precision      : 16-bit
Duration       : 00:03:56.59 = 10433472 samples = 17744 CDDA sectors
File Size      : 41.7M
Bit Rate       : 1.41M
Sample Encoding: 16-bit Signed Integer PCM

I get the following error when trying to connect to my MZ-RH1 with  QHiMDtransfer

Error opening minidisc device (disc image):
Error opening directory “/media/david/disk/HMDHIFI/hmdhifi”: No such file or directory

This error message appears in the terminal window when I start QHiMDTransfer and try to connect to the RH1.

Gtk-Message: 19:47:34.667: GtkDialog mapped without a transient parent. This is discouraged.

I don't know how to invoke QHiMDTransfer with command line switches as I don’t know what the required switches are.

Is there anything else I can try to get a connection or help you in any way to solve this? Thanks.

David.

**Comment by thp** [comment]
Looks like you're trying to open the wrong folder?

    Error opening directory “/media/david/disk/HMDHIFI/hmdhifi”: No such file or directory

Make sure you point it to `/media/david/disk`, not `/media/david/disk/HMDHIFI`.

**Comment by djsbriscoe** [comment]
You are correct. I tried the disk folder and now no error. However, the files on my PC folder are still greyed out in QHiMDTransfer and can't be sent or retrieved from the MD. Is there anything else I can try? Thanks.

**Comment by thp** [comment]
Can you try with MP3 files? Maybe the GUI part of uploading WAV files to
the player isn't implemented yet

djsbriscoe <notifications@github.com> schrieb am Fr., 22. Feb. 2019, 17:48:

**Comment by djsbriscoe** [comment]
Tried a mp3 file (converted from WAV using Audacity). Visible in RH pane of QHiMDTransfer (no longer greyed out like other WAV files). Green left pointing arrow and blue "Upload files to MD" icon are highlighted but on clicking the arrow nothing happens. Clicking on the blue icon brings up a dialog box. I select the mp3 file, click open on the top right and again nothing happens. I restart QHiMDTransfer and then try again but same results.

**Comment by Invictaz** [comment]
@thp I would like to know how this issue proceeds.
